spec/index.bs

```diff
@@ -5521,6 +5521,67 @@ dictionary GPUCommandBufferDescriptor : GPUObjectDescriptorBase {
 
 # Command Encoding # {#command-encoding}
 
+## <dfn interface>GPUCommandsMixin</dfn> ## {#commands-mixin}
+
+{{GPUCommandsMixin}} defines state common to all interfaces which encode commands.
+It has no methods.
+
+<script type=idl>
+interface mixin GPUCommandsMixin {
+};
+</script>
+
+{{GPUCommandsMixin}} adds the following internal slots to interfaces which include it:
+
+<dl dfn-type=attribute dfn-for=GPUCommandsMixin>
+    : <dfn>\[[state]]</dfn>, of type [=encoder state=]
+    ::
+        The current state of the {{GPUCommandEncoder}}, initially set to "[=encoder state/open=]".
```

Comment:
is the reference to `GPUCommandEncoder` here correct? The mixin can be used in a render bundle encoder.